cdap-ui/graphql/schema/applicationSchema.graphql

```diff
@@ -0,0 +1,58 @@
+"""
+ Copyright © 2019 Cask Data, Inc.
+ Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ use this file except in compliance with the License. You may obtain a copy of
+ the License at
+ http://www.apache.org/licenses/LICENSE-2.0
```

Comment:
preserve the white spaces in the license texts

Reply:
Done